backend/common/src/main/java/ai/verta/modeldb/common/config/Config.java

```diff
@@ -36,7 +36,7 @@ public abstract class Config {
   @JsonProperty private boolean event_system_enabled = false;
   @JsonProperty private ServerInterceptor tracingServerInterceptor = null;
   @JsonProperty private ClientInterceptor tracingClientInterceptor = null;
-  @JsonProperty private volatile OpenTelemetry openTelemetry;
+  @JsonProperty private OpenTelemetry openTelemetry;
```

Comment:
This field isn't actually used by anything, anywhere, as far as I can tell. We should remove it.